xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/pom.xml

```diff
@@ -57,6 +57,14 @@
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
     </dependency>
+    <!-- Runtime dependencies -->
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-livetable-ui</artifactId>
+      <version>${project.version}</version>
+      <type>xar</type>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
@mflorea can you confirm it is relevant to add this dependency to make `livedata-livetable` work in a minimal XS distribution (eg, a docker test distrib).
Especially for https://github.com/xwiki/xwiki-platform/blob/e698f4fd5ff0eac8d89ecad761e285af6e6efc2a/xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/java/org/xwiki/livedata/internal/livetable/LiveTableLiveDataEntryStore.java#L135